src/main/java/org/example/api/DeliveryEmployeeService.java

```diff
@@ -3,6 +3,7 @@
 import org.example.cli.DeliveryEmployee;
 import org.example.cli.DeliveryEmployeeProjectRequest;
 import org.example.cli.DeliveryEmployeeRequest;
+import org.example.cli.FailedToUpdateDeliveryEmployeeException;
```

Comment:
this exception isn't being used anywhere?

Reply:
Removed unused exception